src/pybind/mgr/dashboard/services/rgw_client.py

```diff
@@ -175,7 +175,8 @@ def __init__(self,  # pylint: disable-msg=R0913
         self.admin_path = admin_path
 
         s3auth = S3Auth(access_key, secret_key, service_url=self.service_url)
-        super(RgwClient, self).__init__(host, port, 'RGW', ssl, s3auth)
+        # Disable SSL verification to support self-signed certificates.
+        super(RgwClient, self).__init__(host, port, 'RGW', ssl, s3auth, ssl_verify=False)
```

Comment:
Disabling `ssl_verify` per default is not a good idea. An idea would be to make it configurable in a setting.

Reply:
@sebastian-philipp I agree, this should be configurable in some form.